start_esp/start_esp.py

```diff
@@ -569,6 +571,14 @@ def make_argparser():
         action='store_true',
         help=argparse.SUPPRESS)
 
+    parser.add_argument('--client_ip_header', default=None, help='''
+    Defines the request header field whose value will be used to be used as
+    the real client address.
+    {template}.''')
+
+    parser.add_argument('--client_ip_position', default=0, help='''
+    Defines the position of the client IP address. The default value is 0.
+    Same as the array index in many languages, such as Python.''')
```

Comment:
The index usage is the same as ...

Also mention that 

this flag is only applied when --client_ip_header is specified.

Reply:
Done